licenses.d2iq.yaml

```diff
@@ -340,24 +340,12 @@ resources:
       - url: https://github.com/kubernetes-retired/external-storage
         ref: nfs-provisioner-${image_tag}
         license_path: LICENSE
-  - container_image: quay.io/prometheus-operator/prometheus-operator:v0.62.0
-    sources:
-      - url: https://github.com/prometheus-operator/prometheus-operator
-        ref: ${image_tag}
-        license_path: LICENSE
-        notice_path: NOTICE
```

Comment:
This is weird - why are we losing those dependency images. I understand we likely are on a new version, but those new versions should appear here, no?

Reply:
https://github.com/mesosphere/kommander-applications/actions/runs/6147909730/job/16680418101